knudev-liquibase/src/main/resources/db/changelog/team_management/2024_10_11_init_db.yaml

```diff
@@ -31,4 +31,209 @@ databaseChangeLog:
+                  constraints:
+                    nullable: false
 
+  - changeSet:
+      id: create_specialty_table
+      author: DenysLeonenko
+      changes:
+        - createTable:
+            schemaName: team_management
+            tableName: specialty
+            columns:
+              - column:
+                  name: code_name
+                  type: double
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: add_department_id_and_specialty_code_number_columns
+      author: DenysLeonenko
+      changes:
+        - addColumn:
+            schemaName: team_management
+            tableName: account_profile
+            columns:
+              - column:
+                  name: department_id
+                  type: uuid
+                  constraints:
+                    nullable: false
+              - column:
+                  name: specialty_code_name
+                  type: double
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: create_account_department_table
+      author: DenysLeonenko
+      changes:
+        - createTable:
+            schemaName: team_management
+            tableName: account_department
+            columns:
+              - column:
+                  name: account_id
+                  type: integer
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: department_id
+                  type: uuid
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: create_account_specialty_table
+      author: DenysLeonenko
+      changes:
+        - createTable:
+            schemaName: team_management
+            tableName: account_specialty
+            columns:
+              - column:
+                  name: account_id
+                  type: integer
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: specialty_id
+                  type: double
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: create_departments_specialties_table
+      author: DenysLeonenko
+      changes:
+        - createTable:
+            schemaName: team_management
+            tableName: departments_specialties
+            columns:
+              - column:
+                  name: specialties_id
+                  type: double
+                  constraints:
+                    nullable: false
+              - column:
+                  name: departments_id
+                  type: uuid
+                  constraints:
+                    nullable: false
```

Comment:
And also, make department_id as first column, because its more intuitively that department is first

Reply:
done